src/config.js

```diff
@@ -1,5 +1,12 @@
 require("dotenv").config();
 
-const { DB_NAME, DB_USER, DB_PASSWORD, PORT = 3000, DB_HOST } = process.env;
+const {
+  DB_NAME,
```

Comment:
Can we assign default values for all these variables

Reply:
I think yes.